app/Http/Controllers/EntityController.php

```diff
@@ -709,36 +709,44 @@ function setOrUpdateImportedAttributes($entity_id, $row, $headerRow, $attributeI
 
     public function patchAttributes($id, Request $request) {
         $user = auth()->user();
-        if (!$user->can('entity_data_write')) {
+        if(!$user->can('entity_data_write')) {
             return response()->json([
                 'error' => __('You do not have the permission to modify an entity\'s data'),
             ], 403);
         }
 
         try {
             $entity = Entity::findOrFail($id);
-        } catch (ModelNotFoundException $e) {
+        } catch(ModelNotFoundException $e) {
             return response()->json([
                 'error' => __('This entity does not exist'),
             ], 400);
         }
 
         $addedAttributes = [];
-        foreach ($request->request as $patch) {
+
+        if(count($request->request) === 0) {
+            return response()->json([
+                'entity' => $entity,
+                'added_attributes' => $addedAttributes,
+            ]);
```

Comment:
Shall we pass a 204 to inform the client that no data was passed? Maybe good for future debugging purposes.